service/routes.go

```diff
@@ -95,7 +95,12 @@ func driverSchemas() *v1client.Schemas {
 			webhook.ResourceFields[webhookField] = field
 			driverConfig := schemas.AddType(key, value.GetSchema())
 			driverConfig.CollectionMethods = []string{}
+			defaultMap := value.DefaultValues()
```

Comment:
This is close, but I think we need to tweak it. Let's do this:
1. Change the `GetSchemaI()` interface function to `GetDriverConfigResource()`
2. Instead of adding a `DefaultValues` function, add a `CustomizeSchema(schema *v1client.Schema)` and pass it the result of `driverConfig := schemas.AddType(key, value.GetSchema())`
3. In your implementation of that method, have all the logic we currently have in here. So it would look like:
```
driverConfig.CollectionMethods = []string{}
defaultMap := value.DefaultValues()
for k, f := range driverConfig.ResourceFields {
    defaultVal, ok := defaultMap[k]
    if ok {
        f.Default = defaultVal
    }
    f.Create = true
    driverConfig.ResourceFields[k] = f
}
```

Oh, `driverConfig.CollectionMethods = []string{}` can probably stay out in this function where it is automatically done for all drivers.